### ISSUE TITLE
Support managed rule group configs in aws_wafv2_web_acl for the new managed rule AWSManagedRulesATPRuleSet

### ISSUE THREAD

**Issue by lorelei-rupp-imprivata** [hashicorp/terraform-provider-aws#23290]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

A new managed rule group was added for Account Takeover protection. The provider does not seem to support the additional configuration required -- https://docs.aws.amazon.com/waf/latest/APIReference/API_ManagedRuleGroupConfig.html
Please support the additional managed rule group configuration so we can use terraform to set up this rule set.

### New or Affected Resource(s)

* aws_wafv2_web_acl

### Broken Terraform Configuration

```hcl
        statement {
          managed_rule_group_statement {
            name        = "AWSManagedRulesATPRuleSet"
            vendor_name = "AWS"
          }
        }
```
This does not work when you apply because it needs additional configuration

### References

* #0000

**Comment by hasssammalik** [comment]
can we get an update on this one please

**Comment by chrisminton** [comment]
The required configuration looks something like 
```json
      "ManagedRuleGroupConfigs": [
        {
          "LoginPath": string
        },
        {
          "PayloadType": "JSON|FORM_ENCODED"
        },
        {
          "UsernameField": {
            "Identifier": string
          }
        },
        {
          "PasswordField": {
            "Identifier": string
          }
        }
      ]
```

**Comment by y0zg** [comment]
Any update on this please?

**Comment by marianod92** [comment]
Hello everyone!
I consulted in the other [related issue](https://github.com/hashicorp/terraform-provider-aws/issues/23287), I ask again here

Has anyone been able to solve this problem with some workaround through **_aws-cli_** for example?

I tried to activate Account Takeover Prevention through Terraform with a **_null_resource_** and **_aws-cli_**, but I did not find this option in the documentation and reference examples.

Thanks!

**Comment by sidpremkumar** [comment]
@marianod92 were you able to figure this out? Running into the same issue :(

**Comment by marianod92** [comment]
> @marianod92 were you able to figure this out? Running into the same issue :(
> 
> edit: For anyone else this fixed it: #29154

Hey @sidpremkumar

I couldn't get back on this issue yet, but from what I saw in the [Issue #23287](https://github.com/hashicorp/terraform-provider-aws/issues/23287)you can already provision this rule directly from Terraform.
I have not yet been able to test the Terraform provider version > 4.49 to see how the ATP Rule Set behaves.

- https://github.com/hashicorp/terraform-provider-aws/issues/23287#issuecomment-1372896160

On the other hand, in the same issue they commented a solution via aws-cli, here is the link so you can take a look at it:
- https://github.com/hashicorp/terraform-provider-aws/issues/23287#issuecomment-1282501273

Best!
